client/gutenberg/extensions/business-hours/components/hours-list.jsx

```diff
@@ -38,28 +40,32 @@ class HoursList extends Component {
 		this.setState( { data: defaultLocalization }, () => {
 			apiFetch( { path: '/wpcom/v2/business-hours/localized-week' } ).then(
 				data => {
-					this.setState( { localization: data } );
+					this.setState( { localization: data, hasFetched: true } );
 				},
 				() => {
-					this.setState( { localization: defaultLocalization } );
+					this.setState( { localization: defaultLocalization, hasFetched: true } );
 				}
 			);
 		} );
 	}
 
 	render() {
-		const { className, attributes } = this.props;
+		const { className, attributes, edit } = this.props;
 		const { hours } = attributes;
-		const { localization } = this.state;
+		const { localization, hasFetched } = this.state;
 		const { startOfWeek } = localization;
 		return (
 			<dl className={ className }>
-				{ Object.keys( hours )
-					.concat( Object.keys( hours ).slice( 0, startOfWeek ) )
-					.slice( startOfWeek )
-					.map( dayOfTheWeek => {
-						return <HoursRow day={ dayOfTheWeek } data={ localization } { ...this.props } />;
-					} ) }
+				{ hasFetched || ! edit ? (
+					Object.keys( hours )
+						.concat( Object.keys( hours ).slice( 0, startOfWeek ) )
+						.slice( startOfWeek )
+						.map( dayOfTheWeek => {
+							return <HoursRow day={ dayOfTheWeek } data={ localization } { ...this.props } />;
+						} )
+				) : (
+					<p>{ __( 'Loading business hours...' ) }</p>
```

Comment:
We usually use `…` instead of three dots.

Reply:
Probably for a different PR, but I expect we should use a placeholder as a loading state, rather than just some text.